Functions/README.md

```diff
@@ -8,16 +8,15 @@ integration test FirebaseFunctions:
 ### Prereqs
 
 - At least CocoaPods 1.6.0
-- Install [cocoapods-generate](https://github.com/square/cocoapods-generate)
 
```

Comment:
We may take a step further and recommend installing [rvm](https://rvm.io/rvm/install) first. Then we can get rid off `bundle exec` and use just `pod gen FirebaseFunctions.podspec` as before. `rvm` also mitigates the concern about using global Bandler location mentioned by @wilhuff

So the steps will be:
1. Install rvm if not installed
2. `bundle install`
No "CocoaPods 1.6.0" mention will be required in this case since it is controlled by the Gemfile

What do you think?

Reply:
These instructions don't work without `rvm`

```
$ gem install bundle
Fetching: bundle-0.0.1.gem (100%)
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /Library/Ruby/Gems/2.3.0 directory.
```

This is the reason I was suggesting the use of `--path` in the other PR. This is the alternative that works without rvm:
```
sudo gem install bundle
bundle install --path=vendor/bundle
```